src/beast/doc/websocket.qbk

```diff
@@ -309,9 +309,9 @@ and received a close frame.
 
 During read operations, Beast automatically reads and processes control
 frames. Pings are replied to as soon as possible with a pong, received
-pongs are delivered to the pong callback. The receipt of a close frame
-initiates the WebSocket close procedure, eventually resulting in the error
-code [link beast.ref.websocket__error `error::closed`] being delivered
+ping and pongs are delivered to the ping callback. The receipt of a close
```

Comment:
1. How would you word it?

2. Pings and pongs are delivered to the callback to inform the host application that there are "signs of life" on the connection. Typically this will be handled by resetting a timer or setting a flag.

Also note that this already passed review. If you want to suggest changes to Beast please do so in the correct repository:
https://github.com/vinniefalco/Beast

Its not really practical to make changes to Beast here (since that would fork rippled's beast subtree). Practical suggestions in the form of pull requests for improving the docs are **always** welcomed!!!